src/commands/build-util.ts

```diff
@@ -76,22 +98,42 @@ import.meta.hot.dispose(() => {
 }`;
 }
 
-export function wrapHtmlResponse(code: string, hasHmr = false) {
+export function wrapHtmlResponse({
+  code,
+  hasHmr = false,
+  buildOptions,
+}: {
```

Comment:
Another example of ordered params -> option

Reply:
Yea with just JS I'd 100% agree, with TS types I'm ambivalent but overall I'm 👍 if it helps others